Library/Formula/puddletag.rb

```diff
@@ -48,7 +49,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", HOMEBREW_PREFIX/"lib/python2.7/site-packages"
 
     bin.install Dir[libexec/"bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    bin.env_script_all_files(libexec/"bin", :PATH => "/usr/local/bin", :PYTHONPATH => ENV["PYTHONPATH"])
```

Comment:
Don't hardcode `/usr/local`. Not everyone has Homebrew installed there. It should ideally be `"#{bin}"` or if absolutely necessary `"#{HOMEBREW_PREFIX}/bin"`